lib/package-card.coffee

```diff
@@ -187,6 +190,9 @@ class PackageCard extends View
 
   isDisabled: -> atom.packages.isPackageDisabled(@pack.name)
 
+  hasSettings: (pack) ->
+    if atom.config.get(pack.name)? then true else false
```

Comment:
Minor style note here, this line could just be:

```coffee
hasSettings: (pack) ->
    atom.config.get(pack.name)?
```

Ending with a `?` check guarantees a boolean will always be returned.